<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ExportSetPwdDialog</class>
 <widget class="QDialog" name="ExportSetPwdDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>960</width>
    <height>543</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>960</width>
     <height>543</height>
    </rect>
   </property>
   <widget class="QWidget" name="containerWidget" native="true">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>150</y>
      <width>440</width>
      <height>240</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Microsoft YaHei UI Light</family>
     </font>
    </property>
    <widget class="QToolButton" name="cancelBtn">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>170</y>
       <width>120</width>
       <height>36</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Microsoft YaHei UI Light</family>
      </font>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="focusPolicy">
      <enum>Qt::NoFocus</enum>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string>取消</string>
     </property>
    </widget>
    <widget class="QToolButton" name="okBtn">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>170</y>
       <width>120</width>
       <height>36</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Microsoft YaHei UI Light</family>
      </font>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="focusPolicy">
      <enum>Qt::NoFocus</enum>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string>确定</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_9">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>60</y>
       <width>100</width>
       <height>30</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Microsoft YaHei UI Light</family>
       <stylestrategy>PreferDefault</stylestrategy>
      </font>
     </property>
     <property name="text">
      <string>私钥密码：</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLineEdit" name="confirmPwdLineEdit">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>110</y>
       <width>250</width>
       <height>30</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Microsoft YaHei UI Light</family>
      </font>
     </property>
     <property name="contextMenuPolicy">
      <enum>Qt::DefaultContextMenu</enum>
     </property>
     <property name="echoMode">
      <enum>QLineEdit::Password</enum>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>110</y>
       <width>100</width>
       <height>30</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Microsoft YaHei UI Light</family>
       <stylestrategy>PreferDefault</stylestrategy>
      </font>
     </property>
     <property name="text">
      <string>再次输入：</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLineEdit" name="pkPwdLineEdit">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>60</y>
       <width>250</width>
       <height>30</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Microsoft YaHei UI Light</family>
      </font>
     </property>
     <property name="contextMenuPolicy">
      <enum>Qt::DefaultContextMenu</enum>
     </property>
     <property name="echoMode">
      <enum>QLineEdit::Password</enum>
     </property>
    </widget>
    <widget class="QToolButton" name="closeBtn">
     <property name="geometry">
      <rect>
       <x>415</x>
       <y>12</y>
       <width>13</width>
       <height>13</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="focusPolicy">
      <enum>Qt::NoFocus</enum>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <tabstops>
  <tabstop>pkPwdLineEdit</tabstop>
  <tabstop>confirmPwdLineEdit</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
